corehq/apps/reports/dispatcher.py

```diff
@@ -119,6 +121,11 @@ def get_report(self, domain, report_slug, *args):
         """
         return self.get_reports_dict(domain).get(report_slug, None)
 
+    @quickcache(['domain', 'report_slug'], timeout=60)
```

Comment:
any reason not to make this timeout even longer? if i'm reading the code right the only way this can change is during a deploy, no?